src/ui/components/atoms/TabLabel/styles.module.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../../../globals/ads-variables.scss';
+@import url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap');
```

Comment:
please use another approach to embed the font. Use this approach 
https://github.com/Ditmar/OpenScience/blob/main/src/globals/_fonts.scss